### PR TITLE
Remove dependency on react-remarkable

### PR DESCRIPTION
**Pull request by eliask**

### CODE REVIEW

File: webapp/loggedin/appErrors/appErrors.js [hunk 1]
```diff
@@ -31,9 +30,7 @@ const AppError = ({ error, closeAppError }) => {
         ERROR {R.path(['response', 'status'], error)}
       </div>
       <div className="message">
-        <Markdown options={{ html: false }}>
-          {i18n.t(key, params)}
-        </Markdown>
+        {i18nMarkdownToText(i18n, key, params)}
```
Comment (minotogna) [inline webapp/loggedin/appErrors/appErrors.js]: what about have a component in ```app/commonComponents/markdown.js``` to be used like:
```
<Markdown className="message" options={{ html: false }} source={i18n.t(key, params)} />
```

Comment (eliask) [reply webapp/loggedin/appErrors/appErrors.js]: Sure, could do that. The original dependency was only a 30-50 line component in the first place so I opted for this initially.
So just have almost the same API, then?

Comment (minotogna) [reply webapp/loggedin/appErrors/appErrors.js]: yes, the reason is that dependency was not updated anymore, and used old react lifecycle class methods,

**Comment by eliask** [comment]
Hopefully this is sufficient, now.

**Pull request merged**